todo/services/task_service.py

```
@@ -92,31 +97,51 @@ def _validate_pagination_params(cls, page: int, limit: int) -> None:
             raise ValidationError(f"Maximum limit of {PaginationConfig.MAX_LIMIT} exceeded")
 
     @classmethod
-    def _prepare_pagination_links(cls, current_page, page: int, limit: int) -> LinksData:
+    def _prepare_pagination_links(cls, page: int, limit: int, total_count: int, sort_by: str, order: str) -> LinksData:
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)